collectors/carbonblack/collector.js

```diff
@@ -0,0 +1,160 @@
+/* -----------------------------------------------------------------------------
+ * @copyright (C) 2020, Alert Logic, Inc
+ * @doc
+ *
+ * carbonblack class.
+ *
+ * @end
+ * -----------------------------------------------------------------------------
+ */
+'use strict';
+
+const moment = require('moment');
+const PawsCollector = require('@alertlogic/paws-collector').PawsCollector;
+const parse = require('@alertlogic/al-collector-js').Parse;
+const utils = require("./utils");
+
+
+let typeIdPaths = [];
+
+let tsPaths = [];
+
+
+class CarbonblackCollector extends PawsCollector {
+    constructor(context, creds) {
+        super(context, creds, 'carbonblack');
```

Comment:
The newer versions of the super class here can derive the collector type from the context, this constructor is no longer required